### PR TITLE
Updated files for new release and update release-workflow.md to add a missing step

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

* Pushing changes in the files for new release

### List of changes proposed in this PR (pull-request)

* Updated CITATION.cff and .zenodo.json file using the release workflw
* the workflow guide has a missing step that is added in this PR

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: release-workflow.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ To learn more about that, please read the details in [the GitHub documentation](
   - Step 1: Please install cffconvert using the following command: `python3 -m pip install --user cffconvert` (details here: https://pypi.org/project/cffconvert/)
   - Step 2: Run the following command: `cffconvert --validate`
   - Step 3: Once validated, create `.zenodo.json` file using the following command: `cffconvert --format zenodo > .zenodo.json`
-
+  - Step 4: Create the pull request with the change -> review and merge it in the repository
```
Comment (JimMadge) [inline release-workflow.md:17]:
```suggestion
  - Step 4: Create the pull request with the change -> review and merge it in the repository
  
```

Adding a newline here to stop the following paragraph being part of this bullet point.

**Review by JimMadge — CHANGES_REQUESTED** [review]
Looks good to me.

I did think, because the zenodo file can be created from just the `CITATION.cff`, and there is a validation step, this could possibly be done in CI.

Then I found [this workflow](https://github.com/alan-turing-institute/the-turing-way/blob/main/.github/workflows/release-workflow.yml) :smile:.

@sgibson91 @bsipocz is that workflow ready to use? Maybe the instructions could have the workflow method and manual method to make a release?

**Comment by sgibson91** [comment]
> is that workflow ready to use?

I'm not sure, I didn't develop it. I think it would need more investigation so that we are confident in what it's doing.

**Comment by bsipocz** [comment]
I would  suggest to open a separate issue for Jim's comment above about automation, and but otherwise go ahead and merge the PR.

And to answer to the question: I did update some action versions in that workflow config, but haven't done anything about the actual content of it. So yes, it should be tested, I suppose.

**Review by bsipocz — APPROVED** [review]

**Comment by bsipocz** [comment]
btw, on the latest zenodo entry there are some weird things happening with the version numbers (there is a v1.0.3, while here it's v1.0.2, but also there is a v1.0.2 with no changelog, and the changelog link points to diff between v1.0.1..v1.0.2), not sure whether this PR will fix it, but I would suggest to keep an eye on it after this is merged, and open a separate issue if it's still a problem:


Screenshot 2023-03-16 at 17 02 35

**Review by JimMadge — APPROVED** [review]

**Comment by malvikasharan** [comment]
Hi All, the workflow is developed by @timo0thy using the workflow process I wrote. If you can help test the workflow, the manual release might not be needed as that will trigger release periodically.

**Comment by sgibson91** [comment]
@malvikasharan if you can share the documents you wrote, it may help us evaluate this

**Comment by malvikasharan** [comment]
@sgibson91 @timothy22000 wrote this based on this: https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md. I think you helped him in writing this.

**Comment by bsipocz** [comment]
Given the approvals, and that this release have been already made a while back (the tag and release are there already), I go ahead and merge the PR. 

Ideally, there is a zenodo webhook that enables automatic upload from the release, but it's either not there or is not working. Someone with access right might want to double check on it.

**Pull request merged**